lib/bipm/data/outcomes.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "outcomes/body"
+require_relative "outcomes/localized_body"
+require_relative "outcomes/meeting"
+
+module BIPM
+  module Data
+    module Outcomes
+      def self.file_path
+        "#{__dir__}/../../../"
```

Comment:
@webdev778 can we separate the "viewer" (the gem) from the data repository?

Reply:
Actually, let's move the viewer code into bipm-data-importer. That way we have the proper separation.

Reply:
Already done in https://github.com/metanorma/bipm-data-importer/pull/37 , closing.